README.md

```diff
@@ -56,7 +56,7 @@ You are required to use vanilla JavaScript for all parts of this project, includ
 ## Axios
 The only extra JavaScript library we should load is [`Axios`](https://axios-http.com/docs/intro).
 
-To download the `axios` node module, run `npm install`
+To download the `axios` node module, run `yarn install`
 
 To include axios in your project, include the following script tag below the script tag linking `index.js`:
 - `<script src="./node_modules/axios/dist/axios.min.js"></script>`
```

Comment:
Since the initial `index.html` doesn't have the script tag referred to here, it could be confusing. Suggest rewording so that it's clear this will be done during Wave 1 (we don't need to do it yet). Something like:

```suggestion
To include axios in your project, include the following script tag below the script tag linking `index.js`:
- `<script src="./node_modules/axios/dist/axios.min.js"></script>`

This should be done during the Wave 1 initial setup of your `index.html` page.
```